packages/compass-metrics/src/modules/features.js

```diff
@@ -44,6 +44,9 @@ const CollectionResource = FeatureResource.extend({
   eventTrackers: ['stitch'],
   fetched: function(metadata, callback) {
     this._send_event(metadata, callback);
+  },
+  created: function(metadata, callback) {
```

Comment:
why is this necessary?

Reply:
This is how the telemetry module works, you set rules like:

```
{
    registryEvent: 'compass:collection:created',
    resource: 'Collection',
    action: 'created',
    condition: () => true,
    metadata: (version, state) => ({ ... })
  }
```

and then based on resource and action it finds a method in feature.js and checks where to send telemetry based on `eventTrackers` property.